test/inv_test.cpp

```diff
@@ -12,7 +12,7 @@ void set_up_scroll(Item &item, spell_id spell)
 	pcurs = CURSOR_HAND;
 	leveltype = DTYPE_CATACOMBS;
 	Players[MyPlayerId]._pRSpell = static_cast<spell_id>(spell);
-	item._itype = ITYPE_MISC;
+	item._itype = Misc;

```

Comment:
```suggestion
	item._itype = ItemType::Misc;
```

Even if the prefix is optional here it would be best to include it in the test files.

Reply:
I hate these test........... every refactor operation I do in Visual Studio doesn't include them. I'm just not used to not having the tests be part of normal compilation.

Reply:
Then enable the tests... it will compile both the normal build and the tests then.